docs/docsite/rst/scenario_guides/guide_gce.rst

```diff
@@ -254,3 +254,47 @@ Note that use of the "add_host" module above creates a temporary, in-memory grou
 in the 'new_instances' group, if so desired.  Any sort of arbitrary configuration is possible at this point.
 
 For more information about Google Cloud, please visit the `Google Cloud website <https://cloud.google.com>`_.
+
+Migration Guides
+----------------
+
+gce.py -> gcp_compute_instance.py
+`````````````````````````````````
+We're encouraging everyone to move from the `gce` module to the
+`gcp_compute_instance` module. The new `gcp_compute_instance` module has better
+support for all of GCP's features, less dependencies, more flexibility, and
+better supports GCP's authentication systems.
+
+The new `gcp_compute_instance` module supports all of the features of the `gce`
+module (and more!). Below is a mapping of `gce` fields over to
+`gcp_compute_instance` fields.
+
+============================  ==========================================  ======================
+ gce.py                        gcp_compute_instance.py                     Notes 
+============================  ==========================================  ======================
+ state                        status                                      State on gcp_compute_instance is used to describe if the instance exists (present) or does not (absent). Status is used to describe if the instance is on, off, etc.
+ image                        disks[].initialize_params.source_image      You'll need to create a single disk using the disks[] parameter and set it to be the boot disk (disks[].boot = true)
+ image_family                 disks[].initialize_params.source_image      See above.
+ external_projects            disks[].initialize_params.source_image      The name of the source_image will include the name of the project.
+ instance_names               Use a loop or multiple tasks.               Using loops is a more Ansible-centric way of creating multiple instances and gives you the most flexibility.
+ machine_type                 machine_type
```

Comment:
If nothing changes, then do you need to list it?

Reply:
Yeah, I could probably take those off.